test/e2e-go/features/partitionRecovery/partitionRecovery_test.go

```diff
@@ -253,12 +261,16 @@ func TestPartitionHalfOffline(t *testing.T) {
 	a.NoError(err)
 	nc3.FullStop()
 
+	status, err := client.Status()
+	a.NoError(err)
+	roundAfterStop := status.LastRound
+
 	time.Sleep(inducePartitionTime)
 
 	// Get main client to monitor
-	status, err := client.Status()
+	status, err = client.Status()
 	a.NoError(err)
-	a.Equal(waitForRound, status.LastRound, "We should not have made progress since stopping the nodes")
+	a.Equal(roundAfterStop, status.LastRound, "We should not have made progress since stopping the nodes")
```

Comment:
Looks good. Same issue here then? We were getting to `waitForRound+1`?

Reply:
See my explanation above. If it is not clear, I'll explain more.